riff-raff/app/AppComponents.scala

```diff
@@ -188,11 +191,61 @@ class AppComponents(
   val scheduledDeployNotifier =
     new DeployFailureNotifications(config, targetResolver, prismLookup)
 
-  val authAction = new AuthAction[AnyContent](
-    googleAuthConfig,
+  lazy val serviceAccountCredentials = {
+    val getParameterRequest = GetParameterRequest
```

Comment:
This relies on https://github.com/guardian/deploy-tools-platform/pull/881, which has [now been deployed](https://riffraff.gutools.co.uk/deployment/view/d15125ea-a627-4f1a-a159-6de873eeec22).